### PR TITLE
[upload flow] Add Wizard Support

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/47460

## Description
Add support to `Wizard` in the upload flow.

### CODE REVIEW

File: Packs/Wizard/CaseAdoption/caseadoption-wizard.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+	"id": "caseadoption_wizard_sample",
+	"name": "Cade Adoption Wizard Sample",
```
Comment (tkatzir) [inline Packs/Wizard/CaseAdoption/caseadoption-wizard.json:3]:
```suggestion
	"name": "Case Adoption Wizard Sample",
```

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -1638,6 +1639,8 @@ def collect_content_items(self):
                     self._server_min_version = get_updated_server_version(self._server_min_version, content_item,
                                                                           self._pack_name)
 
+                    if self._pack_name == 'Wizard':
+                        logging.info(content_item.get('name', 'DeanLog: couldn\'t find the name :('))
```
Comment (tkatzir) [inline Tests/Marketplace/marketplace_services.py:1643]: Sanitize the word "Dean"

**Review by tkatzir — CHANGES_REQUESTED** [review]
Looking good.
Maybe the name "Wizard" is too generic?

File: Packs/Malware/pack_metadata.json [hunk 3]
```diff
@@ -78,4 +78,4 @@
         "xsoar",
         "marketplacev2"
     ]
-}
\ No newline at end of file
+}
```
Comment (tkatzir) [inline Packs/Malware/pack_metadata.json:81]: Why does it show as changed?

File: Tests/Marketplace/marketplace_constants.py [hunk 4]
```diff
@@ -203,7 +204,8 @@ def pack_displayed_items(cls):
             PackFolders.GENERIC_DEFINITIONS.value, PackFolders.GENERIC_FIELDS.value, PackFolders.GENERIC_MODULES.value,
             PackFolders.GENERIC_TYPES.value, PackFolders.LISTS.value, PackFolders.JOBS.value,
             PackFolders.PARSING_RULES.value, PackFolders.MODELING_RULES.value, PackFolders.CORRELATION_RULES.value,
-            PackFolders.XSIAM_DASHBOARDS.value, PackFolders.XSIAM_REPORTS.value, PackFolders.TRIGGERS.value
+            PackFolders.XSIAM_DASHBOARDS.value, PackFolders.XSIAM_REPORTS.value, PackFolders.TRIGGERS.value,
+            PackFolders.WIZARD.value
```
Comment (tkatzir) [inline Tests/Marketplace/marketplace_constants.py:208]:
```suggestion
            PackFolders.WIZARD.value,
```

File: Tests/Marketplace/marketplace_constants.py [hunk 5]
```diff
@@ -221,7 +223,7 @@ def json_supported_folders(cls):
             PackFolders.WIDGETS.value, PackFolders.GENERIC_DEFINITIONS.value, PackFolders.GENERIC_FIELDS.value,
             PackFolders.GENERIC_MODULES.value, PackFolders.GENERIC_TYPES.value, PackFolders.LISTS.value,
             PackFolders.PREPROCESS_RULES.value, PackFolders.JOBS.value, PackFolders.XSIAM_DASHBOARDS.value,
-            PackFolders.XSIAM_REPORTS.value, PackFolders.TRIGGERS.value
+            PackFolders.XSIAM_REPORTS.value, PackFolders.TRIGGERS.value, PackFolders.WIZARD.value
```
Comment (tkatzir) [inline Tests/Marketplace/marketplace_constants.py:226]:
```suggestion
            PackFolders.XSIAM_REPORTS.value, PackFolders.TRIGGERS.value, PackFolders.WIZARD.value,
```

File: Tests/Marketplace/marketplace_services.py [hunk 6]
```diff
@@ -1610,6 +1610,7 @@ def collect_content_items(self):
                 PackFolders.XSIAM_DASHBOARDS.value: "xsiamdashboard",
                 PackFolders.XSIAM_REPORTS.value: "xsiamreport",
                 PackFolders.TRIGGERS.value: "trigger",
+                PackFolders.WIZARD.value: "wizard"
```
Comment (tkatzir) [inline Tests/Marketplace/marketplace_services.py:1613]:
```suggestion
                PackFolders.WIZARD.value: "wizard",
```

File: Packs/Malware/pack_metadata.json (hunk 3 above)
Comment (DeanArbel) [reply Packs/Malware/pack_metadata.json:81]: My `ide` automatically adds `newline` to the end of files (previously there was none). I should remove it, disregard the change.

File: Tests/Marketplace/marketplace_constants.py [hunk 7]
```diff
@@ -192,6 +192,7 @@ class PackFolders(enum.Enum):
     XSIAM_DASHBOARDS = 'XSIAMDashboards'
     XSIAM_REPORTS = 'XSIAMReports'
     TRIGGERS = 'Triggers'
+    WIZARD = 'Wizards'
```
Comment (tkatzir) [inline Tests/Marketplace/marketplace_constants.py:195]: Seems like the standard is:
```suggestion
    WIZARDS = 'Wizards'
```

**Review by tkatzir — APPROVED** [review]

**Pull request merged**